extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-config.js

```diff
@@ -67,88 +77,54 @@ describes.realWin(
-            },
-          ],
-          'aggregateRating': {
-            'ratingValue': 4.4,
-            'reviewCount': 89,
-            'reviewUrl': 'https://www.google.com',
-          },
-        },
-        {
-          'productUrl': 'https://www.google.com',
-          'productId': 'flowers',
-          'productTitle': 'Flowers',
-          'productVendor': 'Very Long Flower Company Name',
-          'productPrice': 10.0,
-          'productPriceCurrency': 'USD',
-          'productIcon':
-            '/examples/visual-tests/amp-story/img/shopping/icon.png',
-          'productImages': [
-            {
-              'url': 'https://source.unsplash.com/SavQfLRm4Do/500x500',
-              'alt': 'flowers',
-            },
-          ],
-          'aggregateRating': {
-            'ratingValue': 4.4,
-            'reviewCount': 89,
-            'reviewUrl': 'https://www.google.com',
-          },
-        },
       ],
     };
 
     const keyedDefaultInlineConfig = {
       'lamp': defaultInlineConfig.items[0],
       'art': defaultInlineConfig.items[1],
-      'chair': defaultInlineConfig.items[2],
-      'flowers': defaultInlineConfig.items[3],
     };
 
     beforeEach(async () => {
-      pageElement = env.win.document.createElement('amp-story-page');
-      pageElement.id = 'page1';
+      win = env.win;
+      storeService = getStoreService(win);
+      registerServiceBuilder(win, 'story-store', function () {
+        return storeService;
+      });
+      pageElement = <amp-story-page id="page1"></amp-story-page>;
+      win.document.body.appendChild(pageElement);
+
+      shoppingAttachment = win.document.createElement(
+        'amp-story-shopping-attachment'
+      );
+      shoppingAttachment.setAttribute('layout', 'nodisplay');
+      const story = win.document.createElement('amp-story');
+      win.document.body.appendChild(story);
+      story.appendChild(pageElement);
+      pageElement.appendChild(shoppingAttachment);
     });
 
     async function createAmpStoryShoppingConfig(
-      src = undefined,
+      src = null,
       config = defaultInlineConfig
     ) {
-      const shoppingAttachment = env.win.document.createElement(
-        'amp-story-shopping-attachment'
-      );
-      shoppingAttachment.setAttribute('layout', 'nodisplay');
       shoppingAttachment.setAttribute('src', src);
       shoppingAttachment.appendChild(
         <script type="application/json">{JSON.stringify(config)}</script>
       );
-      const story = env.win.document.createElement('amp-story');
-      env.win.document.body.appendChild(story);
-      story.appendChild(pageElement);
-
-      pageElement.appendChild(shoppingAttachment);
       return getShoppingConfig(shoppingAttachment);
     }
 
+    it('should build shopping config component', async () => {
+      expect(() => getShoppingConfig(pageElement)).to.not.throw();
+    });
+
     it('throws on no config', async () => {
       expectAsyncConsoleError(async () => {
```

Comment:
Looking at [expectAsyncConsoleError()](https://github.com/ampproject/amphtml/blob/458d1d52f05988f497970ff15548d67d0e9639c4/testing/console-logging-setup.js#L107), it looks like its parameter should be the error string instead of a function. So the usages of `expectAsyncConsoleError()` in this file will need to be fixed. I think you should be able to use [this test](https://github.com/ampproject/amphtml/blob/458d1d52f05988f497970ff15548d67d0e9639c4/extensions/amp-story-auto-ads/0.1/test/test-story-ad-page.js#L337-L343) as a template.